actionview/lib/action_view/rendering.rb

```diff
@@ -49,6 +49,15 @@ def _helpers
       end
 
       def build_view_context_class(klass, supports_path, routes, helpers)
+        if superclass.respond_to?(:view_context_class) &&
```

Comment:
Should we extract all those conditionals to a private method? That way we don't need the comment.

Reply:
👍 done